saas/views/saas_view.xml

```diff
@@ -66,6 +66,33 @@
         </field>
     </record>
 
+    <!-- SaaS Module -->
+    <record id='saas_module_view_tree' model='ir.ui.view'>
+        <field name="name">saas.module.tree</field>
+        <field name="model">saas.module</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="description" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="saas_module_form_view" model="ir.ui.view">
+        <field name="name">saas.module.form.view</field>
+        <field name="model">saas.module</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                        <field name="description"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
```

Comment:
Make separate files for each type of views (saas_template_views.xml, saas_module_views.xml, etc). Keep ``saas_views.xml`` for top menu only:


```


    <!-- Menu items-->
    <menuitem name="SaaS"
              id="saas_main_menu"/>
```